python/MOCK_AIS.py

```diff
@@ -69,13 +71,26 @@ def make_ros_message(self):
             self.id,
             self.lat,
             self.lon,
-            self.headingDegrees,
+            (90.0 -self.headingDegrees) % 360,
```

Comment:
Yeah I see that conversion lots of places in our code. May look into making that change you talked about :)